docs: adapt docs to new story (#5800)
jina-ai/jina: README.md

  - Improved engineering efficiency thanks to the Jina AI ecosystem, so you can focus on innovating with the data applications you build.
  - Free CPU/GPU hosting via [Jina AI Cloud](https://cloud.jina.ai).

> Jina value proposition can be found quite similar to FastAPI, however there are some fundamental differences

Reviewer: ```suggestion
Jina's value proposition may seem quite similar to that of FastAPI. However, there are several fundamental differences:
```

>  **Data structure & Communication Protocols**

Reviewer: ```suggestion
 **Data structure and communication protocols**
```

  - FastAPI communication relies on Pydantic and Jina relies on [DocArray](https://github.com/docarray/docarray) allowing Jina to support multiple protocols
  to expose its services.

 **Advanced orchestration and scaling capabilities**

>   - Jina allows to deploy applications formed by multiple microservices that can be containerized and scaled independently.

Reviewer: ```suggestion
  - Jina lets you deploy applications formed from multiple microservices that can be containerized and scaled independently.
```

  - Jina allows you to easily containerize and orchestrate your services easily providing concurrency and scalability.

 **Journey to the cloud**

>   - Jina aims to provide a smooth transition from your local development using [DocArray](https://github.com/docarray/docarray) to your local serving using Jina orchestration layer

Reviewer: ```suggestion
  - Jina provides a smooth transition from local development (using [DocArray](https://github.com/docarray/docarray)) to local serving using (Jina's orchestration layer)
```
